src/stories/SystemFields.stories.ts

```diff
@@ -0,0 +1,57 @@
+import type { Meta, StoryObj } from '@storybook/vue3';
+import SystemFields from '@components/smart/SystemFields.vue';
+
+const meta: Meta<typeof SystemFields> = {
+  title: 'Smart components/SystemFields',
```

Comment:
Misschien hier `Smart components/System-fields` van maken zodat de documentatie nav items in hetzelfde formaat zijn. (of de Adres-crab title aanpassen)

Reply:
Ik zou dan de Adres-crab naar AdresCrab veranderen. Als we in de applicaties components gaan gebruiken zal het ook <AdresCrab> zijn denk ik en niet <Adres-crab>

Reply:
beejte merge conflicten maken ofzo? zie maar dat mijnen branch eerst gemerged wordt :laughing: